test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/Internal/EnumerableWrapperProviderTest.cs

```diff
@@ -79,17 +80,12 @@ public void Ignores_NullInstances()
         [InlineData(typeof(PersonList))]
         public void ThrowsArugmentExceptionFor_ConcreteEnumerableOfT(Type declaredType)
         {
-            // Arrange
-            var expectedMessage =
-                "The type must be an interface and must be or derive from 'IEnumerable`1'." +
-                $"{Environment.NewLine}Parameter name: sourceEnumerableOfT";
-
             // Act and Assert
             var ex = Assert.Throws<ArgumentException>(() => new EnumerableWrapperProvider(
```

Comment:
Totally unrelated to your change, but could you fix up the indentation on this line while you're touching this line of code?